exercises/concept/tim-from-marketing/.docs/introduction.md

```diff
@@ -7,8 +7,8 @@ In Java, the `null` literal is used to denote the absence of a value.
 Primitive data types in java all have a default value and therefore can never be `null`.
```

Comment:
This is outside your change, but suggest capitalizing the J in Java for consistency. 

```suggestion
Primitive data types in Java all have a default value and therefore can never be `null`.
```